website/docs/cloud-docs/api-docs/feature-sets.mdx

```diff
@@ -156,211 +170,15 @@ curl \
         "sso": true,
         "private-networking": true,
         "no-code-modules": true,
+        "run-task-limit": null,
+        "run-task-workspace-limit": null,
+        "run-task-mandatory-enforcement-limit": null,
+        "policy-set-limit": null,
+        "policy-limit": null,
+        "policy-mandatory-enforcement-limit": null,
+        "versioned-policy-set-limit": null,
         "sentinel": true
       }
-    },
-    {
```

Comment:
Note - I got rid of most of the feature sets here because 1. it just made the page HUGE for no reason. 2. There's no extra information here, so the reader doesn't lose out anything with this being deleteld.